iam/api-client/service_account_keys.py

```diff
@@ -22,23 +22,31 @@
 """
 
 import argparse
+# [START iam_create_key]
+# [START iam_list_keys]
+# [START iam_delete_key]
 import os
 
 from google.oauth2 import service_account
 import googleapiclient.discovery
 
-credentials = service_account.Credentials.from_service_account_file(
-    filename=os.environ['GOOGLE_APPLICATION_CREDENTIALS'],
-    scopes=['https://www.googleapis.com/auth/cloud-platform'])
-service = googleapiclient.discovery.build(
-    'iam', 'v1', credentials=credentials)
+# [END iam_create_key]
+# [END iam_list_keys]
+# [END iam_delete_key]
 
 
 # [START iam_create_key]
 def create_key(service_account_email):
     """Creates a key for a service account."""
 
     # pylint: disable=no-member
+    credentials = service_account.Credentials.from_service_account_file(
```

Comment:
It seems strange that we have to define this at all since Cloud Client libraries handle this automatically. Are we certain that discovery is the right library to use and that discovery can't be coaxed to do this automatically?

Reply:
I was hoping it would have a Cloud library somewhere but maybe it doesn't. At any rate, we can assume that's accurate for now. Thanks.